app/models/package_manager/maven/jboss_ea.rb

```diff
@@ -1,18 +0,0 @@
-# frozen_string_literal: true
-
-class PackageManager::Maven::JbossEa < PackageManager::Maven::Common
```

Comment:
I would probably hold off on removing these files until we can take a look through the data and verify we don't have any `Versions` pointing to these repository sources